src/mod.js

```diff
@@ -265,5 +265,5 @@ function euclideanMod (numerator, denominator) {
   return result < 0 ? result + denominator : result
 }
 
-module.exports = v
-v.Vec3 = Vec3
+export { Vec3, v }
+export default v
```

Comment:
Why move index.js to mod.js ?

Reply:
`index.js` is a lie that comes from front-end http servers. You should know that the “dir import” (import * from './') is not in the ECMAScript spec, which is why a lot of people agree on `mod.js`/`mod.ts` as the entry point of a module.